suite/tests/client-interface/annotation-concurrency.appdll.c

```diff
@@ -59,7 +59,6 @@ EXPORT void
 jacobi_init(int matrix_size, bool enable_annotations)
 {
     invoke_annotations = enable_annotations;
-
```

Comment:
Best to revert this to avoid superfluous unrelated noise in the diff.

Reply:
Right, I didn't notice that, thanks. I reverted this change.